app/assets/stylesheets/style.css.scss

```diff
@@ -131,14 +178,42 @@ fieldset.tagline {
   font-family: Lato;
 }
 
-#neighborhood h1 {
-  font-family: Lato;
+#neighborhood .one_neighborhood {
+  width: 100%;
 }
 
-#neighborhood_info {
-  padding-top: 4em;
+#neighborhood h1 {
+  font-family: Lato;
 }
 
 #ratings_list {
   padding-top: 3em;
 }
+
+#contributors {
+  border: #B23875 solid .2em;
+  border-radius: .4em;
+  margin-top: 4em;
+  margin-bottom: 4em;
+  height: 20em;
+  background-image: url("boston.png");
+}
+
+#contributors p {
+  text-align: center;
+  font-size: 3em;
+  font-family: Pacifico;
+  color: #000000;
```

Comment:
Color literals like `#000000` should only be used in variable declarations; they should be referred to via variable everywhere else.